lib/auth_app/accounts/user.ex

```diff
@@ -37,9 +30,8 @@ defmodule AuthApp.Accounts.User do
   """
   def registration_changeset(user, attrs, opts \\ []) do
```

Comment:
Check create_user_registration or similar in the context. It is still passing options not used here. We can also probably unify email_changeset and registration_changeset, removing more code.